### PR TITLE
Update readme file for Message Reactions

### PR DESCRIPTION
**Pull request by Peter-Maguire**

### CODE REVIEW

**Comment by artismarti** [comment]
@Peter-Maguire There's a couple of small changes we made to the READMEs earlier today. Take a look here: 
https://github.com/ably-labs/atomic-examples/blob/bdbc95151ddf6bef8938c047c56e0d9f4807fa6a/src/components/AvatarStack/README.md

**Review by artismarti — CHANGES_REQUESTED** [review]

File: src/components/MessageReactions/README.md [hunk 1]
```diff
@@ -1 +1,37 @@
-## Message Reaction
+## Overview
+
+This repository contains the code for Message Reactions - a demo leveraging Ably's [message interactions](https://ably.com/docs/realtime/messages#message-interactions) feature to add emoji reactions to messages.
```
[the remark this answers is not in the document]
Comment (ben-pickering) [reply src/components/MessageReactions/README.md:3]: contains the code for Message Reactions -> contains the code for Emoji Reactions

**Comment by mohyour** [comment]
> Looks good, but can we rename the files/folders/components to say "Emoji Reactions" instead of "Message Reactions"? I thought Peter had done that already, but maybe it's not pushed yet. https://ably-real-time.slack.com/archives/C03P7344H1P/p1660060176722079

Yes. The change is on another branch which is currently in review

**Comment by artismarti** [comment]
Can you point me to the branch/PR, please @mohyour ?
I can't find it. Sorry! 🙈

**Comment by mohyour** [comment]
> Can you point me to the branch/PR, please @mohyour ? I can't find it. Sorry! 🙈

https://github.com/ably-labs/atomic-examples/pull/18

**Comment by artismarti** [comment]
oh of course! Sorry, I got misled by the title of the PR.

**Review by artismarti — APPROVED** [review]

**Pull request merged**